File: 03-ordenacao/C#/Ordenacao/OrdenacaoSegundaChave2/Pessoa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using static System.Net.Mime.MediaTypeNames;

namespace OrdenacaoSegundaChave2
{
    internal class Pessoa
    {
        public string Nome { get; set; }    
        public int Idade { get; set; }

        public Pessoa(string nome, int idade)
        {
            Nome = nome;
            Idade = idade;
        }

        public override string ToString()
        {
            return "Pessoa{" + "nome=" + Nome + ", idade=" + Idade + '}';
        }
    }
}

File: 03-ordenacao/Java/OrdenacaoSegundaChave/src/Pessoa.java

public class Pessoa {
    public String nome;
    public int idade;
    
    public Pessoa(String nome, int idade) {
        this.nome = nome;
        this.idade = idade;
    }   

    @Override
    public String toString() {
        return "Pessoa{" + "nome=" + nome + ", idade=" + idade + '}';
    }
    
    
}
